docs/items/item-components.md

```diff
@@ -199,6 +199,24 @@ Note: The "all" enchantable slot allows you to apply any enchantment that you wa
 | sword         |
 | all           |
 
+### Enchantability Value
+Determines the item's enchantability, influencing the quality and quantity of potential enchantments. Higher values boost the chance of guaranteeing more powerful enchantments. The table below details enchantability scores across different materials, showcasing their ability to get enchantments.
+
+| Material  | Armor Enchantability | Sword/Tool Enchantability |
+|-----------|----------------------|---------------------------|
+| Wood      | N/A                  | 15                        |
+| Leather   | 15                   | N/A                       |
+| Stone     | N/A                  | 5                         |
+| Chain     | 12                   | N/A                       |
+| Iron      | 9                    | 14                        |
+| Gold      | 25                   | 22                        |
+| Diamond   | 10                   | 10                        |
+| Turtle    | 9                    | N/A                       |
+| Netherite | 15                   | 15                        |
+| Other     | 1                    | 1                         |
+
+For an in-depth exploration of enchantability and its impact on the game, refer to the [Enchanting Mechanics on the Unofficial Minecraft Wiki](https://minecraft.fandom.com/wiki/Enchanting_mechanics#Enchantability).
```

Comment:
Update to new Minecraft Wiki URL: https://minecraft.wiki/w/Enchanting_mechanics#Enchantability